### ISSUE TITLE
Accessibility md-select aria-labelledby

### ISSUE THREAD

**Issue by Lindsor** [angular/material2#7015]
#### Bug:
aria-labelledby shows up in DOM even if it has no value

#### What is the expected behavior?
aria-labelledby should never be empty, if there is no labelledby value then it should not be in the DOM

#### What is the current behavior?
aria-labelledby shows up in the DOM even if empty

#### What are the steps to reproduce?
http://embed.plnkr.co/cLXRzvn2uWQ5y5YmiU8L/
inspect select, notice empty aria-labelledby

#### What is the use-case or motivation for changing an existing behavior?
Currently throws accessibility issues

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
"@angular/material": "2.0.0-beta.8"

**Comment by crisbeto** [comment]
Closing since the associated PR was merged.
